src/App.tsx

```diff
@@ -1,61 +1,135 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { FormEvent, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [selectedUser, setSelectedUser] = useState('-1');
+  const [userError, setUserError] = useState(false);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [todos, setTodos] = useState(todosFromServer);
+  const [titleLanguageError, setTitleLanguageError] = useState(false);
+
+  const titleHandler = (event: FormEvent<HTMLInputElement>) => {
```

Comment:
Function's name should start with a verb (so you could easily tell what your method actually do)